bundles/org.openhab.binding.speedtest/README.md

```diff
@@ -0,0 +1,48 @@
+# speedtest Binding
+
+This binding simplifies the addition of network speed tests from your openHAB instance.   Simply put, it's a wrapper around Ookla's SpeedTest ( https://www.speedtest.net/apps/cli ).  
```

Comment:
```suggestion
This binding simplifies the addition of network speed tests from your openHAB instance.
Simply put, it's a wrapper around Ookla's SpeedTest ( https://www.speedtest.net/apps/cli ).  
```

Please put each sentence on its own line, this makes future changes much more clear and quicker to review. Please go through and change any more times this occurs.

Reply:
done